### ISSUE TITLE
Upgrade to MongoDB 4.0

### ISSUE THREAD

**Issue by klaussner** [meteor/meteor-feature-requests#269]
The main feature of MongoDB 4.0 (scheduled to release next summer) will be **multi-document ACID transactions**: https://www.mongodb.com/transactions.

**Comment by mitar** [comment]
So what is the process here? MongoDB4 4.0 is supported, but transactions are not yet, no?

**Comment by mitar** [comment]
Minimongo API? You mean server-side MongoDB API.

**Comment by mitar** [comment]
But you are right that it would be in fact interesting to have transactions also in Minimongo. This could allow one to prepare multiple changes to documents without reactivity triggering and then commit and re-render at once.

**Comment by klaussner** [comment]
Yeah, I meant both. :smile: I think it would be good to keep the API as isomorphic as possible.

**Comment by a4xrbj1** [comment]
So what's the latest on this one? This isn't encouraging news 

> **MongoDB 4.0 is supported in Meteor 1.8 but transactions are not (yet) part of the Minimongo API (related FR: #311)**

It goes along the same line of what I wrote in the State of Javascript thread. I don't get the feeling that MDG is supporting Meteor **enough**, in the way that **too much focus (meaning manpower)** is on GraphQL development and important features (that would make Meteor way more attractive to most serious developers) like the above multiple ACID transactions features isn't implemented.

That leads to small Startups and companies giving up their hope for Meteor. At least that's how my employees and me see it.

MongoDb makes big steps forward (like everyone agrees to in the announcement of MongoDb 4 thread) but Meteor is further falling behind and no clear timeline or commitment to implement this feature is given by MDG.
